utils/ALA_dialogues.cfg

```diff
@@ -814,7 +813,7 @@
     [/message]
     [message]
         speaker=Geobard
-        message= _ "Woe coming from the river, commander!"
+        message= _ "Woe, they are coming from the river, commander!"
```

Comment:
Explain this one?

Reply:
A subject is missing.

Reply:
I thought "woe" was being used as the subject there

Reply:
Woe is for me rather a call like _Damn it_ or _For heaven_.

Reply:
If woe should be the subject (nagas are enboded as woe here), the sentence should sound: Woe is coming from the river, commander!

Reply:
How about "There's woe coming to us from the river, commander!"?

Reply:
OK

Reply:
OK I went with that in [b434ef3](https://github.com/cooljeanius/A_Little_Adventure/pull/23/commits/b434ef320ac406f56ae8cd7051c917522be5d9de)